www/js/components/point-map.js

```diff
@@ -203,4 +190,7 @@ function select(state) {
     mapState: state.mapState
   };
 }
+
+PointMap.defaultProps = { onLeafletMove: noOps, afterMoved: noOps };
```

Comment:
I think if you say `<Map onLeafletMove={undefined} />`,  no listener will be created. I don't think we should give the `Map` event callbacks that aren't needed.